README.md

```diff
@@ -0,0 +1,107 @@
+
+# 📚📙📖 Online Bookstore API 📖📙📚
+This is an application for online book store . 
+An Online Bookstore API is a back-end part of virtual store  where customers can browse the catalog and select books of interest. User can select many books and those books stored in cart. At checkout time, the items in the shopping cart will be presented as an order.
+Admins can manage this online bookstore.  This application serves as the core infrastructure , delivering crucial functionalities for book management, user authentication, and order processing.
+
+ ### 🛠️ Technologies and Tools Used
+- Spring Boot: For building the application framework.
+- Spring Security: To handle authentication and authorization.
+- Spring Data JPA: For database interactions using JPA.
+- Spring Web: For building web-based applications with Spring MVC.
+- Swagger: For API documentation and testing.
+- Hibernate: As the ORM tool.
+- MySQL: For the relational database.
+- Liquibase: For database schema versioning and management.
+- Docker: For containerizing the application.
+- Maven: For project management and dependency management.
+### 🚀 Current Functionalities
+- ##### Authentication 
+  - Register a new user: POST /api/auth/register
+  - Authenticate a user: POST /api/auth/login
+- ##### Book
+  - Get books from catalog: GET /api/books
+  - Get book for id: GET /api/books/{id}
+  - Create a new book: POST /api/books
+  - Update a specific book: PUT /api/books/{id}
+  - Delete a specific book: DELETE /api/books/{id}
+- ##### Category
+  - Get all books' categories: GET /api/categories
+  - Get a category by id: GET /api/categories/{id}
+  - Get all books by current category: GET /api/categories/{id}/books
+  - Add a new category (only for role "ADMIN"): POST /api/categories
+  - Update a current category (only for role "ADMIN"):PUT /api/categories/{id}
+  - Delete a current category (only for role "ADMIN"): DELETE /api/categories/{id}
+- ##### Shopping Cart
+  - Get a user's shopping cart: GET /api/cart
+  - Add book to the shopping cart: POST /api/cart
+  - Update quantity of a book in the shopping cart: PUT /api/cart/cart-items/{cartItemId}
+  - Remove a book from the shopping cart: DELETE /api/cart/cart-items/{cartItemId}
+- ##### Orders
+  - Create an order: POST /api/orders
+  - Get history of orders: GET /api/orders
+  - Get all OrderItems for a specific order: GET /api/orders/{orderId}/items
+  - Get a specific OrderItem within an order: GET /api/orders/{orderId}/items/{itemId}
+  - Update order status: PATCH /api/orders/{id}
+- #####  Swagger Documentation
+  - the API documentation at http://localhost:8080/swagger-ui.html.
+  - You can explore all available endpoints, their descriptions, request parameters, and response schemas.
+
+### Live Preview 💻
+Check out the live demo of the Book Store application:
+
+- [Live Demo](https://www.loom.com/share/9b5f2e7bd2f845c2ad2a25361530810f)
```

Comment:
as of the demo video: if you are located in Poland and planning on looking for a job in Poland as well - I think it makes sense to record a video in English or Polish, especially if you have a title and timecodes in polish

Reply:
Thank you )